webapp/channels/src/packages/mattermost-redux/src/selectors/entities/users.ts

```diff
@@ -502,7 +503,7 @@ export function makeSearchProfilesInChannel() {
 }
 
 export function searchProfilesInCurrentChannel(state: GlobalState, term: string, skipCurrent = false): UserProfile[] {
-    const profiles = filterProfilesStartingWithTerm(getProfilesInCurrentChannel(state), term);
+    const profiles = filterProfilesStartingWithTermOrPositionMatchingWithTerm(getProfilesInCurrentChannel(state), term);
```

Comment:
I decided that it would be best to isolate this solution to just the position field so we do not cause any regression of  the functionality of the other fields.